examples/plot_mdm_spd.py

```diff
@@ -0,0 +1,59 @@
+"""Illustration example for the MDM classification algorithm in dimension 2."""
+
+
+import geomstats.datasets.sample_sdp_2d
+import geomstats.visualization as visualization
+from geomstats.geometry.spd_matrices import SPDMatrices
+from geomstats.geometry.spd_matrices import SPDMetricAffine
+from geomstats.learning.mdm import RiemannianMinimumDistanceToMeanClassifier
+
+
+def main():
+    """Execute illustration of MDM supervised classifier."""
+    n_samples = 100
+    n_features = 2
+    n_classes = 3
+
+    # generate toy dataset of 2D SPD matrices
+    dataset_generator = geomstats.datasets.sample_sdp_2d.DatasetSPD_2D(
+        n_samples, n_features, n_classes)
+    data, labels = dataset_generator.generate_sample_dataset()
+
+    # plot dataset as ellipses
+    ellipsis = visualization.Ellipsis2D()
+    for i in range(n_samples):
+        x = data[i]
+        y = dataset_generator.data_helper.get_label_at_index(i, labels)
+        ellipsis.draw(x, color=ellipsis.colors[y], alpha=.1)
+
+    # define and fit MDM classifier to data
+    metric = SPDMetricAffine(n=n_features)
+    MDMEstimator = RiemannianMinimumDistanceToMeanClassifier(
+        metric, point_type='matrix')
+    MDMEstimator.fit(data, labels)
+
+    # plot Frechet means computed in the MDM
+    for i in range(n_classes):
+        ellipsis.draw(
+            MDMEstimator.G[i],
```

Comment:
`.G` notation is not very explicit + uppercase letters names are reserved for classes in Python. Maybe `.estimate_`?

Reply:
Done.